Documentation generate separately (#3914)
ESMValGroup/ESMValTool: doc/sphinx/source/index.rst

routine comparison of models and observations. It includes a large collection of community recipes and observation data formatters to CMOR standards.

**ESMValCore** is a software package which provides the core functionality for ESMValTool. It is a workflow to find CMIP data, and apply
commonly used pre-processing functions.

To get a first impression of what ESMValTool and ESMValCore can do for you,
have a look at our blog posts
`Analysis-ready climate data with ESMValCore <https://blog.esciencecenter.nl/easy-ipcc-powered-by-esmvalcore-19a0b6366ea7>`_
and
`ESMValTool: Recipes for solid climate science <https://blog.esciencecenter.nl/esmvaltool-recipes-for-solid-climate-science-da5b33814f69>`_.

Basic documentation schema
--------------------------
This gives a brief idea of topics in each of the packages to help find information. As ESMValTool encompasses ESMValCore there will be some overlap between them.
For more detailed information, see documentation navigation to the left.

.. container::
   :name: figarch

   .. figure:: figures/ESMValSchemaDiagram.png
      :alt: Brief topics for tool and core.

>       :figclass: align-center

Author: I was thinking its more a condensed diagram to assist in understanding with the broader topics but these topics might need to be reviewed and refined, I'm not sure there would be appropriate specific links to them all

>       :figclass: align-center

Reviewer: I still think it would be nicer to generate it dynamically from the documentation pages, but if that's technically too challenging, it's fine to leave it as is.

Welcome to ESMValTool's documentation!
======================================

**ESMValTool** is a community diagnostics and performance metrics tool for the evaluation of Earth System Models (ESMs) that allows for
routine comparison of models and observations. It includes a large collection of community recipes and observation data formatters to CMOR standards.

**ESMValCore** is a software package which provides the core functionality for ESMValTool. It is a workflow to find CMIP data, and apply
commonly used pre-processing functions.

To get a first impression of what ESMValTool and ESMValCore can do for you,
have a look at our blog posts
`Analysis-ready climate data with ESMValCore <https://blog.esciencecenter.nl/easy-ipcc-powered-by-esmvalcore-19a0b6366ea7>`_
and
`ESMValTool: Recipes for solid climate science <https://blog.esciencecenter.nl/esmvaltool-recipes-for-solid-climate-science-da5b33814f69>`_.

Basic documentation schema
--------------------------
This gives a brief idea of topics in each of the packages to help find information. As ESMValTool encompasses ESMValCore there will be some overlap between them.

> For more detailed information, see documentation navigation to the left. Please also see
> `ESMValCore documentation <https://docs.esmvaltool.org/projects/ESMValCore/en/latest/index.html>`_. ESMValCore can be used without ESMValTool such as in Jupyter

Reviewer 2: Use inclusive language that doesn't refer to "seeing" an element:
```suggestion
The documentation navigation to the left provides more detailed information about ESMValTool.
`ESMValCore documentation <https://docs.esmvaltool.org/projects/ESMValCore/en/latest/index.html>`_ is also available. ESMValCore can be used without ESMValTool such as in Jupyter
```

   <iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?si=pUXrXB8C8bLRfQHY&amp;list=PLFjfi2xLaFpJp59LvDc1upQsj_xzFlFLc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>

How to contribute
-----------------
Refer to ESMValTool :ref:`contributing to the community<community>` for a guide on contributing recipes and diagnostics.

Refer to ESMValCore :ref:`Contributing <contributing>`  for information on contributing code.

Get in touch!
-------------

> Contact information is available :ref:`here <Support-and-Contact>`.

Author: Good idea! 
So you mean each of the 3 links? (ESMValTool contributing, ESMValCore contributing, Support and contact)

> Contact information is available :ref:`here <Support-and-Contact>`.

Reviewer 2: This change would also make the main page less busy 😊

> Contact information is available :ref:`here <Support-and-Contact>`.

Author: Thanks @ehogan can you share the code please? I think there could also be a version thing going on as it doesn't look quite the same as what the theme documentation is showing

> Contact information is available :ref:`here <Support-and-Contact>`.

Reviewer 2: I opened a new issue for this (#4215) to allow this PR to be merged 😊
